### DISCUSSION TITLE
Observable list not updating in UI/View

### DISCUSSION THREAD

**Discussion by ashrafmozhikkal** [mobxjs/mobx.dart#764] [General]
For a dynamic/Multi type observable list, the data is not updating in UI. But value is updated inside list for sure.

See my code below:

 @observable
  ObservableList<TaxDetails> totalTaxAmountList =
      ObservableList<TaxDetails>.of([]);

**Comment by ashrafmozhikkal** [comment]
If we add a new data into list, its updating. No problem. But when we update existing list, it is not updating in view. Kindly assist please.

**Comment by deadsoul44** [comment]
This is a typical issue with mobx and should be in docs. As far as I understand, the reason is that the element itself is not a store and is not reactive. You can try replaceRange method to update the list.
